packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -12,21 +12,15 @@
 import android.graphics.Rect;
 import android.graphics.RectF;
 import android.graphics.Typeface;
+import com.caverock.androidsvg.SVG;
 
 import androidx.core.content.res.ResourcesCompat;
 
 public class CozyMarkerBuilder {
-    private final int shadowSize = 4;
```

Comment:
we are removing these because we're passing through `size` param in `CozyMarkerData` right

Reply:
Actually we don't have shadow anymore, and others were determined dynamically for the legacy marker. But now we just have the new marker which is fixed, that's why I removed.

But regarding to size, yeah there is a property in contract related to that, just didn't implement yet because we are not using yet.